wazo_agid/modules/user_get_vmbox.py

```diff
@@ -42,6 +42,8 @@ def user_get_vmbox(agi: FastAGI, cursor: DictCursor, args: list[str]) -> None:
     agi.set_variable('XIVO_VMMAIN_OPTIONS', vmmain_options)
     agi.set_variable('XIVO_MAILBOX', user.vmbox.mailbox)
     agi.set_variable('XIVO_MAILBOX_CONTEXT', user.vmbox.context)
+    if user.vmbox.password:
+        agi.set_variable('XIVO_VM_PASSWORD', user.vmbox.password)
```

Comment:
It's a new variable, no? In this case, can we use `WAZO_VM_PASSWORD` instead :sweat_smile: